### ISSUE TITLE
Making label field optional in Input elements.

### ISSUE THREAD

**Issue by anuragagarwal561994** [marmelab/admin-on-rest#788]
Currently for my use case, I don't require input elements to have labels on them. The reason being, I am using these Input Elements as part of my List and want to edit them from my list only. For that I made a wrapper component around them which does the job of sending the update response, but the only issue is label element which does not look nice in the list section.

Right now I have to make exactly similar Input elements which does not have label fields to solve this issue.

There can be other such cases when these labels are not required by user.

**Comment by djhi** [comment]
Have you tried to add `addLabel={false}` on the inputs ? See the [doc](https://marmelab.com/admin-on-rest/Inputs.html#writing-your-own-input-component). I haven't tried yet but it might be enough

**Comment by fzaninotto** [comment]
Editing from the list isn't a feature provided by the core admin-on-rest. I'm happy that you somehow manage to do it with the existing components, but you'll probably reach limits that you will only overcome by using your own components.

**Comment by anuragagarwal561994** [comment]
@djhi No `addLabel` is not available by default, I had tried it before posting. In the example given on the doc for creating custom Input, it is present.

@fzaninotto Yes I agree with you. I was following the review-page in the demo. Sometimes there are very minor changes we require to change, like accept and reject buttons on review page. For such boolean or select scenarios, it becomes difficult for the end user to first go to edit page -> edit -> save and then redirect. For my current solution I made a component called `EditableWrapper` below which I am making EditableInputFields for each type like it is present in AOR with material-ui components. EditableWrapper handles the job of sending CRUD_UPDATE when element value is changed.
